spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'spec_helper'
+
+describe SessionsController do
+
+  describe 'GET :new' do
+    context 'user is not logged-in' do
+      it 'renders a new template' do
+        get :new
+        expect(response).to render_template :new
```

Comment:
This should be the Rails default behavior so I would not test this. Kevin is trying to show testing all at the beginning, but later on he says default Rails should not be tested - also anything you have not written should not be tested.